dog.html

```diff
@@ -0,0 +1,195 @@
+              <h3>Jake</h3>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+              <a onclick="addTotal('123.45')">Adopt</a>
+            </div>
+          </section>
+
+          <section>
+            <button onclick="showInfo('Angus', 'Mix', '$123.45')">
+              <img src="images/angus-card.jpg" class="dog-pic" alt="angus">
+            </button>
+            <div class="profile-info">
+              <h3>Angus</h3>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+              <a onclick="addTotal('123.45')">Adopt</a>
+            </div>
+          </section>
+
+          <section>
+            <button onclick="showInfo('Violet', 'Mix', '$123.45')">
+              <img src="images/violet-card.jpg" class="dog-pic" alt="violet">
+            </button>
+            <div class="profile-info">
+              <h3>Violeet</h3>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+              <a onclick="addTotal('123.45')">Adopt</a>
+            </div>
+          </section>
+
+          <section>
+            <button onclick="showInfo('Piper', 'Mix', '$123.45')">
+              <img src="images/piper-card.jpg" class="dog-pic" alt="piper">
+            </button>
+            <div class="profile-info">
+              <h3>Piper</h3>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+              <a onclick="addTotal('123.45')">Adopt</a>
+            </div>
+          </section>
+
+          <section>
+            <button onclick="showInfo('Maximus', 'Mix', '$123.45')">
+              <img src="images/maximus-card.jpg" class="dog-pic" alt="maximus">
+            </button>
+            <div class="profile-info">
+              <h3>Maximus</h3>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+              <a onclick="addTotal('123.45')">Adopt</a>
+            </div>
+          </section>
+
+          <section>
+            <button onclick="showInfo('Luna', 'Mix', '$123.45')">
+              <img src="images/luna-card.jpg" class="dog-pic" alt="luna">
+            </button>
+             <div class="profile-info">
+              <h3>Luna</h3>
+              <p>Cost to Adopt: $123.45</p>
+              <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+              <a onclick="addTotal('123.45')">Adopt</a>
```

Comment:
this looks good

Reply:
I wonder know is there any way to do this part?